src/editor/components/form/select.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+interface SelectProps {
+  label: string,
+  options: any,
```

Comment:
For options it would make sense to have an interface.
Could look something like this:

```ts
interface Option {
  value: any
  name: string
}
```

I'd choose `any` as the type for the option value because it could also be a number or something else and the functionality will stay the same no matter the type.

Reply:
good suggestion